secrets/secrets.go

```
@@ -0,0 +1,59 @@
+package secrets
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+// smClient used for mock testing.
+type smClient interface {
+	GetSecretValue(input *secretsmanager.GetSecretValueInput) (*secretsmanager.GetSecretValueOutput, error)
+}
+
+// AWSSecrets supports the GetSecretValue method.
+type AWSSecrets struct {
+	client smClient
+}
+
+var defaultAWSSecrets = getInstance()
+
+func getInstance() *AWSSecrets {
+	// Should this take dependency on 'env' package and call env.AwsRegion()?
+	region := os.Getenv("AWS_REGION")
```

Don't think we should use the env package here, so as we can keep the different packages separate.

---

Curious as to what will happen when AWS_REGION is not set.
It will default to "" but nothing will fail until line 31?
Is there somewhere we can throw an error?
Maybe we should add a default to us-west-2 since odds are this will be the env?